app/components/DateRangeSelector.vue

```diff
@@ -0,0 +1,154 @@
+<template>
+  <v-card class="pa-4 ma-4" elevation="2">
+    <v-card-title class="text-h6 pb-2">Date Range Filter</v-card-title>
+    <v-row>
+      <v-col cols="12" sm="4">
+        <v-text-field
+          v-model="fromDate"
+          label="From Date"
+          type="date"
+          variant="outlined"
+          density="compact"
+          @update:model-value="updateDateRange"
+        />
+      </v-col>
+      <v-col cols="12" sm="4">
+        <v-text-field
+          v-model="toDate"
+          label="To Date"
+          type="date"
+          variant="outlined"
+          density="compact"
+          @update:model-value="updateDateRange"
+        />
+      </v-col>
+      <v-col cols="12" sm="4" class="d-flex align-center">
```

Comment:
buttons are not aligned with date dropdowns

<img width="2517" height="185" alt="Image" src="https://github.com/user-attachments/assets/fee7e60c-6b59-423c-95c0-f14e7e4c5ff7" />